client/my-sites/customer-home/cards/tasks/domain-upsell/style.scss

```diff
@@ -7,9 +10,12 @@ $illustration-width: 214px;
 	&.task__domain-upsell {
 		.task__illustration {
 			display: flex;
-			width: calc(45% + 32px);
-			margin-bottom: -32px !important;
-			margin-right: -32px !important;
+			
+			@include break-small() {
```

Comment:
Let's use `@include break-small` instead of  `@include break-small()` to be more consistent with the rest of the codebase. Same for the rest of the instances in the PR!